xmip/preprocessing.py

```diff
@@ -27,7 +27,7 @@ def cmip6_renaming_dict():
         "x": ["i", "ni", "xh", "nlon"],
         "y": ["j", "nj", "yh", "nlat"],
         "lev": ["deptht", "olevel", "zlev", "olev", "depth"],
-        "bnds": ["bnds", "axis_nbounds", "d2"],
+        "bnds": ["axis_nbounds", "d2"],
```

Comment:
@jbusecke this is the line I mentioned, so from the `cmip6_renaming_dict`, you were renaming `bnds` to `bnds`.  But okay, I'll revert [a6a5835](https://github.com/jbusecke/xMIP/pull/300/commits/a6a5835d1836f04177b70d5b44f2e6f0a8a2753a) - maybe that is safer.